"An empty interface is equivalent to `{}`" lint error on new empty project

### Describe the bug

The SvelteKit demo app application with TypeScript enabled fails the `npm run lint` phase:
```
$ npm run format
...
$ npm run lint

> my-app@0.0.1 lint
> prettier --ignore-path .gitignore --check --plugin-search-dir=. . && eslint --ignore-path .gitignore .

Checking formatting...
All matched files use Prettier code style!

/home/ufo/workspaces/temp/my-app/src/app.d.ts
  10:12  error  An empty interface is equivalent to `{}`  @typescript-eslint/no-empty-interface
  12:12  error  An empty interface is equivalent to `{}`  @typescript-eslint/no-empty-interface
  14:12  error  An empty interface is equivalent to `{}`  @typescript-eslint/no-empty-interface

/home/ufo/workspaces/temp/my-app/src/lib/form.ts
  68:15  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any

✖ 4 problems (3 errors, 1 warning)
```

### Reproduction

- Create a new project from scratch with: `npm init svelte@next my-app`
- Select "SvelteKit demo app", Typescript `yes`, ESLint `yes`, Prettier `yes`
- run `npm run format`
- run `npm run lint`


### Logs

_No response_

### System Info

```shell
System:
    OS: Linux 5.11 Ubuntu 21.04 (Hirsute Hippo)
    CPU: (8) x64 Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz
    Memory: 6.41 GB / 31.31 GB
    Container: Yes
    Shell: 5.1.4 - /bin/bash
  Binaries:
    Node: 16.13.1 - ~/Programs/node/bin/node
    Yarn: 1.22.17 - ~/Programs/node/bin/yarn
    npm: 8.1.2 - ~/Programs/node/bin/npm
  Browsers:
    Chrome: 98.0.4758.80
    Chromium: 98.0.4758.80
    Firefox: 96.0
  npmPackages:
    @sveltejs/adapter-auto: next => 1.0.0-next.17 
    @sveltejs/kit: next => 1.0.0-next.265 
    svelte: ^3.46.0 => 3.46.4
```


### Severity

serious, but I can work around it

### Additional Information

_No response_

--- a/.changeset/great-seahorses-fry.md
+++ b/.changeset/great-seahorses-fry.md
@@ -1,0 +1,5 @@
+---
+'create-svelte': patch
+---
+
+fix `@typescript-eslint/no-empty-interface` lint error when starting a new app with eslint

--- a/packages/create-svelte/templates/default/src/app.d.ts
+++ b/packages/create-svelte/templates/default/src/app.d.ts
@@ -7,9 +7,9 @@ declare namespace App {
 		userid: string;
 	}
 
-	interface Platform {}
+	// interface Platform {}
 
-	interface Session {}
+	// interface Session {}
 
-	interface Stuff {}
+	// interface Stuff {}
 }

--- a/packages/create-svelte/templates/skeleton/src/app.d.ts
+++ b/packages/create-svelte/templates/skeleton/src/app.d.ts
@@ -4,10 +4,10 @@
 // for information about these interfaces
 declare namespace App {
 	// interface Locals {}
-	//
+
 	// interface Platform {}
-	//
+
 	// interface Session {}
-	//
+
 	// interface Stuff {}
 }

--- a/packages/create-svelte/templates/skeleton/src/app.d.ts
+++ b/packages/create-svelte/templates/skeleton/src/app.d.ts
@@ -3,11 +3,11 @@
 // See https://kit.svelte.dev/docs/types#the-app-namespace
 // for information about these interfaces
 declare namespace App {
-	interface Locals {}
-
-	interface Platform {}
-
-	interface Session {}
-
-	interface Stuff {}
+	// interface Locals {}
+	//
+	// interface Platform {}
+	//
+	// interface Session {}
+	//
+	// interface Stuff {}
 }

